ratenyu/users/templates/users/profile.html

```diff
@@ -3,21 +3,47 @@
 {% block content %}
 {% load static %}
 
-<div class = "user-panel">
+<div class = "user-panel" id="profile_div">
     <h1 class = "user-details">Hello <br><br> {{user_details.user}}!</h1>
     <div class="user-form">
         <h4 class = 'user-form'> Name </h4>
-        <p> {{user_details.name}} </p>
+        <p id='user_name'> {{user_details.name}}</p>
         <h4 class = 'user-form'> Username </h4>
-        <p> {{user_details.user}} </p>
+        <p id="user_username"> {{user_details.user}} </p>
         <h4 class = 'user-form'> Major </h4>
-        <p> {{user_details.major}} </p>
+        <p id="user_major" > {{user_details.major}} </p>
         <h4 class = 'user-form'> Student Status </h4>
-        <p> {{user_details.student_status}} </p>
-            <button type="submit" class="user-form">Edit Profile</button>
+        <p id="user_status"> {{user_details.student_status}} </p>
+        <button type="submit" class="user-form" onclick="hideProfile(document.querySelectorAll('.user-panel'))" id="edit_profile">Edit Profile</button>
     </div>
 </div>
 
+<form method="POST">
+    {% csrf_token %}
+    <div hidden class = "user-panel" id="edit_profile_div">
+        <h1 class = "user-details">Hello <br><br> {{user_details.user}}!</h1>
+        <div class="user-form">
+            <label for="user_name_input" class = 'user-form'> Name </label>
+            <input class="edit-form" id='user_name_input' name="user_name_input" value=" {{user_details.name}}"/>
+            <label for="user_username_input" class = 'user-form'> Username </label>
+            <input readonly class="edit-form" id="user_username_input" name="user_username_input" value="{{user_details.user}}"/>
+            <label for="user_major_input" class = 'user-form'> Major </label>
+            <input class="edit-form" id="user_major_input"  name="user_major_input" value="{{user_details.major}}"/>
+            <label for="user_status_input" class = 'user-form'> Student Status </label>
+            <select name="user_status_input" class="edit-dropdown" required="" id="user_status_input">
+                <option value="freshman">Freshman</option>
```

Comment:
Can you make the option values Title case too? Like "Freshman" instead of "freshman

Reply:
not really, thats the cause for the data not save into the database. They should match the value in the user_detail. One way to solve it is to change the user_detail.

Reply:
On the UI, the student status is displayed in lowercase, if we can change it to titlecase then it would be great

Reply:
Got it, just made the change. I'll let you know after done with the dropdown list